src/common/m_derived_types.fpp

```diff
@@ -122,9 +122,7 @@ module m_derived_types
 
         type(ic_model_parameters) :: model !< Model parameters
 
-        real(kind(0d0)) :: epsilon, beta !<
-        !! The isentropic vortex parameters administrating, respectively, both
-        !! the amplitude of the disturbance as well as its domain of influence.
+        real(kind(0d0)) :: epsilon, beta
```

Comment:
what are these variables still used for? i think the might still be used elsewhere. can you document it like in the doxygen hook before?

Reply:
They appear to be spherical harmonics parameters - I have documented them as such.

Reply:
you need the `!<` part